packages/contracts/test/CollSurplusPool.js

```diff
@@ -87,33 +94,6 @@ contract('CollSurplusPool', async accounts => {
     await th.assertRevert(borrowerOperations.claimCollateral({ from: A }), 'CollSurplusPool: No collateral available to claim')
   })
 
-  it("CollSurplusPool: claimColl(): Reverts if owner cannot receive ETH surplus", async () => {
```

Comment:
why have you deleted this test?

Reply:
this tests verfied that the address claiming their collaterals can receive eth, but now we are sending SOV tokens, so any address can receive them